gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -31,9 +31,12 @@
 #include <gnuradio/sincos.h>
 #include <boost/format.hpp>
 
+// Used to convert division to multiplication for speed
+#define CL_ONE_OVER_2PI 0.15915494309189533577
```

Comment:
You might consider moving this constant into gr/math.h ... seems like something that might be used elsewhere too!